client/src/store/historyStore/model/watchHistory.js

```diff
@@ -21,6 +21,16 @@ let lastUpdateTime = null;
 // last time changed history items have been requested
 let lastRequestDate = new Date();
 
+document.addEventListener("visibilitychange", function () {
+    if (document.visibilityState === "visible") {
+        // Poll every 3 seconds when visible
+        throttlePeriod = 3000;
+    } else {
+        // Poll every 60 seconds when hidden/backgrounded
+        throttlePeriod = 60000;
```

Comment:
How about not polling at all? Since we are going to poll after bringing the tab to the foreground. Although this could potentially accumulate too many changes?

Reply:
That'd be fine with me too, but you can set up notifications for completed jobs to appear in the tab title, so I suppose those wouldn't work. I wouldn't be sad to break that though if it means I don't have to regularly kill my chrome 😆

Reply:
Just dropping to 60 seconds is a 20x reduction; maybe start there? I like the idea of backing off on browser idle too, we could add that in later as we see how this behaves (and hopefully helps!).